packages/@uppy/utils/src/ErrorWithCause.test.ts

```diff
@@ -1,5 +1,5 @@
 import { describe, expect, it } from 'vitest'
-import ErrorWithCause from './ErrorWithCause.js'
+import ErrorWithCause from './ErrorWithCause.ts'
```

Comment:
This currently gives an error, even after adjusting the `tsconfig` as I suggested:

```
1. An import path can only end with a '.ts' extension when 'allowImportingTsExtensions' is enabled. [5097]
```

Not sure how to solve that one. Regardless it's probably better to import from `lib` in JS tests?

Reply:
No, importing from lib has always been a bad idea, we do it sometimes to workaround limitations of Jest, but Vitest doesn't have that. IMO the DX is much better if we can run the tests without the need to build.